js/cryptocom.js

```diff
@@ -587,13 +587,20 @@ module.exports = class cryptocom extends Exchange {
         if (limit !== undefined) {
             request['page_size'] = limit;
         }
-        const [ marketType, query ] = this.handleMarketTypeAndParams ('fetchOrders', market, params);
-        const method = this.getSupportedMapping (marketType, {
+        const [ marketType, marketTypeQuery ] = this.handleMarketTypeAndParams ('fetchOrders', market, params);
+        let method = this.getSupportedMapping (marketType, {
             'spot': 'spotPrivatePostPrivateGetOrderHistory',
             'margin': 'spotPrivatePostPrivateMarginGetOrderHistory',
             'future': 'derivativesPrivatePostPrivateGetOrderHistory',
             'swap': 'derivativesPrivatePostPrivateGetOrderHistory',
         });
+        const [ marginMode, query ] = this.handleMarginModeAndParams ('fetchOrders', marketTypeQuery);
+        if (marginMode !== undefined) {
+            if (marginMode !== 'cross') {
```

Comment:
@Dan-krm @samgermain 
For exchanges that support only the basic cross margin, do you think supporting this could be beneficial?

```
const isMargin = this.safeValue(params, 'margin', false)
```
I'm suggesting this because these exchanges might not use cross/isolated terms when there's only one available (probably they'll refer only to "margin"), so I think it could make the user's life easier. Let me know your thoughts

Reply:
If we going to have unified usage across exchanges, then when need to stick with the `marginMode` schema. I think that if only one marginMode is supported though, that we should overwrite `handleMarginModeAndParams` in the subclass and do the check there. 

We can also check for type `margin` on these exchanges, but that will become outdated if this exchange adds the other marginMode

Reply:
@Dan-krm Overwrite `handleMarginModeAndParams` in `cryptocom` and do the check in the overwritten method

Reply:
Let me know your feedback on how I overwrite handleMarginModeAndParams

Reply:
> We can also check for type margin on these exchanges, but that will become outdated if this exchange adds the other marginMode

I don't like this, type is tied to a different concept

> If we going to have unified usage across exchanges, then when need to stick with the marginMode schema
@samgermain I was not saying to ditch `marginMode` I was suggesting supporting both